docs/user_guide/HOWTO/Configure_Ssh.rst

```diff
@@ -1,127 +1,121 @@
-``ssh::client``, configured with reasonable defaults for the OS and environment:
+By default, SIMP will include the ``ssh`` class from the ``simp-ssh`` module
+under most of the :ref:`SIMP scenarios <simp scenarios>`.  If it is absent
+from the scenario you have selected, add the following to :term:`Hiera` to
+include the class:
 
-.. code-block:: puppet
+.. code-block:: yaml
 
-   include 'ssh'
+   simp::classes:
+     - 'ssh'
 
-.. NOTE::
+To exclude the SIMP ``ssh`` class so that you can manage SSH configuration
+by other means, refer to :ref:`Disable SSH Management <disable_ssh_management>`.
+
+The SIMP ``ssh`` class is configured to automatically include the
+``ssh::server`` and ``ssh::client`` classes. These classes, in turn, manage
+the SSH daemon settings for incoming connections and the SSH client settings
+for outgoing connections, respectively. They configure SSH with reasonable
+defaults for the OS and environment.
 
-   The examples below feature ``include 'ssh::server'`` and ``include
-   'ssh::client'``, but most SIMP scenarios already include them both via
-   ``ssh``.  So, for SIMP systems, you will customize parameter settings
-   for ``ssh::server`` and ``ssh::client`` via Hiera.
+If you want to disable SIMP management of either the SSH server or client
+settings, you can do so through Hiera.  For example, to disable SIMP management
+of both:
+
+.. code-block:: yaml
+
+   ssh::enable_client: false
+   ssh::enable_server: false
 
 
 Managing Settings for the SSH Server
 ------------------------------------
 
-Including ``ssh::server`` with the default options will manage the server with
-reasonable settings for each host's environment.
+The ``ssh::server`` class is responsible for configuring ``sshd``.  It delegates
+configuration of specific ``sshd`` settings found in ``sshd_config`` to
+``ssh::server::conf``.  Detailed descriptions of the these settings are provided
+in the ``sshd_config(5)`` man page.
 
+The ``ssh::server::conf`` class uses a number of SIMP-specific parameters, such
+as whether the system is FIPS enabled, has a firewall, or utilizes LDAP. This
+allows seamless integration of SSH with other SIMP-managed applications in the
+larger SIMP enviornment.  For example, ``ssh::server::conf`` ensures the
+appropriate fallback ciphers are used, ensures proper authentication is
+configured, and allows SSH traffic through the firewall.  More detailed
+information is provided in the ``simp-ssh`` module README file.
 
 Configuring ``ssh::server::conf`` from Hiera
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+``ssh::server::conf`` provides default configuration for key ``sshd_config``
+settings and a mechanism to set most other settings.
+
 To customize the SSH server, edit the parameters of ``ssh::server::conf`` using
-:term:`Hiera` or :term:`ENC`.
+Hiera or :term:`ENC`.
 
 .. NOTE::
 
    Unlike many SIMP modules, these customizations cannot be made
    directly with a resource-style class declaration―they *must* be
```

Comment:
To prevent the reader from confusing an _em-dash_ with a hyphen, I would add spaces around it, so:
```
   directly with a resource-style class declaration ― they *must* be
```

Reply:
Adding some (non-actionable) meta-nitpickery to @pillarsdotnet's:  

- This is very good practice for em dashes that will be displayed in fixed-width fonts (like comments in source code).   In fact, many linters suggest simply using hypens instead.
- However: this is not a concern with RestructuredText paragraph text.  Once rendered, readthedocs will display em dashes at the paragraph's full character width—which, at three 3–4 times the length of a hyphen ('—' vs '-'), is unlikely to confuse readers.